### PR TITLE
feat: setup company fixtures

### PR DESCRIPTION
**Pull request by MyuddinKhatri**
Issue - #10 
Setup company fixtures for Companies - Ambrosia Pie Company, Chelsea Fruit Co, and Quincy Cloudberry Farm.

### CODE REVIEW

**Review by Alchez — APPROVED** [review]

File: test_utils/fixtures/setup.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[
```
Comment (agritheory) [inline test_utils/fixtures/setup.json:1]: I think it's more appropriate to call this file `company.json`

File: test_utils/utils/setup_fixtures.py [hunk 2]
```diff
@@ -14,6 +16,35 @@ def get_fixtures_data_from_file(filename):
...
+		if company == setup.get("company_name"):
+			setup.update(
+				{
+					"fy_start_date": today.replace(month=1, day=1).isoformat(),
+					"fy_end_date": today.replace(month=12, day=31).isoformat(),
+				}
+			)
+			setup_complete(setup)
+			frappe.db.commit()
+	enable_all_roles_and_domains()
```
Comment (agritheory) [inline test_utils/utils/setup_fixtures.py:38]: I think we want to remove both `enable_all_roles_and_domains()` and `set_defaults_for_tests()` - they don't really apply to how we do testing and if they're required we can handle it in the project(s).

**Review by agritheory — CHANGES_REQUESTED** [review]
A couple of tweaks here. I think we will have a bit of a "we won't know until we do the downstream work" problem here and that's OK.

**Review by Alchez — APPROVED** [review]

File: test_utils/utils/setup_fixtures.py [hunk 3]
```diff
@@ -14,6 +15,33 @@ def get_fixtures_data_from_file(filename):
...
+		if company == setup.get("company_name"):
+			setup.update(
+				{
+					"fy_start_date": today.replace(month=1, day=1).isoformat(),
+					"fy_end_date": today.replace(month=12, day=31).isoformat(),
+				}
+			)
+			setup_complete(setup)
+			frappe.db.commit()
+	frappe.db.commit()
```
Comment (agritheory) [inline test_utils/utils/setup_fixtures.py:37]: This `commit` isn't required

**Review by agritheory — APPROVED** [review]

**Pull request merged**